framework/utils.py

```diff
@@ -32,6 +32,32 @@ def is_internal_ip(ip):
     parsed_ip = IPAddress(ip)
     return parsed_ip.is_private
 
+def get_rank(user_rank, owtf_rank):
```

Comment:
@saganshul ptp does not have `CRITICAL` by design, since it deemed that only user could have the context to rank an issue as `CRITICAL`.

`PASSING` does not exist, since ptp does not evaluate if a report is passing or not. However, `UNKNOWN` is used when no ranking could be found. It should be noted though that `UNKNOWN` is `0`, while OWTF uses `0` for `PASSING`.

See https://github.com/owtf/ptp/blob/develop/ptp/libptp/constants.py